<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>signup</class>
 <widget class="QMainWindow" name="signup">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1088</width>
    <height>567</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1088</width>
    <height>567</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1088</width>
    <height>567</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>signup</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>wfIcon.ico</normaloff>wfIcon.ico</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="minimumSize">
    <size>
     <width>1088</width>
     <height>567</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>1088</width>
     <height>567</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #191F32;</string>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>584</x>
      <y>108</y>
      <width>339</width>
      <height>325</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>wardenfort 1.png</pixmap>
    </property>
   </widget>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>345</x>
      <y>10</y>
      <width>407</width>
      <height>538</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QFrame {
    border-radius: 25px; 
    background-color: rgba(44, 60, 75, 0.5); 
}</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>488</x>
      <y>21</y>
      <width>103</width>
      <height>77</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(44, 60, 75, 0); </string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../../../../Downloads/7-removebg-preview 1.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>418</x>
      <y>-1</y>
      <width>241</width>
      <height>228</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(44, 60, 75, 0); </string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>wardenfort (1) 2.png</pixmap>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>383</x>
      <y>144</y>
      <width>334</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Inter</family>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit
 {
    border-radius: 10px; 
    background-color: rgba(44, 60, 75, 1); 
	padding: 14px;
	color: #FFFFFF;
}</string>
    </property>
    <property name="text">
     <string>First Name</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_2">
    <property name="geometry">
     <rect>
      <x>383</x>
      <y>212</y>
      <width>334</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Inter</family>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit
 {
    border-radius: 10px; 
    background-color: rgba(44, 60, 75, 1); 
	padding: 14px;
	color: #FFFFFF;
}</string>
    </property>
    <property name="text">
     <string>Last Name</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_3">
    <property name="geometry">
     <rect>
      <x>383</x>
      <y>280</y>
      <width>334</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Inter</family>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit
 {
    border-radius: 10px; 
    background-color: rgba(44, 60, 75, 1); 
	padding: 14px;
	color: #FFFFFF;
}</string>
    </property>
    <property name="text">
     <string>Username</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_4">
    <property name="geometry">
     <rect>
      <x>383</x>
      <y>353</y>
      <width>334</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Inter</family>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit
 {
    border-radius: 10px; 
    background-color: rgba(44, 60, 75, 1); 
	padding: 14px;
	color: #FFFFFF;
}</string>
    </property>
    <property name="text">
     <string>Email</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_5">
    <property name="geometry">
     <rect>
      <x>383</x>
      <y>424</y>
      <width>334</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Inter</family>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit
 {
    border-radius: 10px; 
    background-color: rgba(44, 60, 75, 1); 
	padding: 14px;
	color: #FFFFFF;
}</string>
    </property>
    <property name="text">
     <string>Password</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>685</x>
      <y>434</y>
      <width>20</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(44, 60, 75, 0); </string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>View_hide_lightsss.png</pixmap>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>492</y>
      <width>275</width>
      <height>34</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Inter</family>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    border-radius: 15px; 
    background-color: rgba(22, 27, 46, 0.5);
	color: #0097B2; 

}
QPushButton:hover { background-color: #374151; } QPushButton:pressed { background-color: #2C3345;} </string>
    </property>
    <property name="text">
     <string>Create Account</string>
    </property>
   </widget>
   <widget class="QToolButton" name="toolButton">
    <property name="geometry">
     <rect>
      <x>46</x>
      <y>21</y>
      <width>40</width>
      <height>34</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QToolButton { border: none; background: transparent; }</string>
    </property>
    <property name="text">
     <string>...</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>Expand_down_light.png</normaloff>Expand_down_light.png</iconset>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1088</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
